core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -437,11 +438,21 @@ class GroupMetadataManager(val brokerId: Int,
 
         while (currOffset < highWaterMark && !shuttingDown.get()) {
           buffer.clear()
-          val fileRecords = log.read(currOffset, config.loadBufferSize, maxOffset = None, minOneMessage = true)
-            .records.asInstanceOf[FileRecords]
-          val bufferRead = fileRecords.readInto(buffer, 0)
+          // We only add the READ_UNCOMMITTED parameter here because EasyMock in the GroupMetadataManagerTest
+          // cannot seem to deal with the default value for isolationLevel in log.read() -- it throws an
+          // IllegalStateException.
```

Comment:
Seems like passing `isolationLevel` explicitly is a good idea anyway. Maybe we don't need this comment.